docs/sources/operators-guide/reference-http-api/index.md

```diff
@@ -903,3 +906,59 @@ GET /compactor/ring
 ```
 
 Displays a web page with the compactor hash ring status, including the state, healthy and last heartbeat time of each compactor.
+
+### Start block upload
+
+```
+POST /api/v1/upload/block/{block}
+```
+
+Starts the uploading of a TSDB block with a given ID to object storage. The client should send the block's
```

Comment:
We should document that `meta.json` must have use Thanos` extension with list of files. `mimirtool` does this automatically, but if someone writes their own uploader, they may need this information.

Reply:
Done, PTAL.